src/CompletionProvider.php

```diff
@@ -156,8 +154,12 @@ public function provideCompletion(PhpDocument $doc, Position $pos): CompletionLi
                     }
                 }
             } else {
-                $fqn = $this->definitionResolver->resolveReferenceNodeToFqn($node);
-                $prefixes = $fqn !== null ? [$fqn] : [];
+                $prefix = $this->definitionResolver->resolveReferenceNodeToFqn($node);
+                $index = ($index = strpos($prefix, '->')) ? $index : strpos($prefix, '::');
```

Comment:
`$prefix` can be `null`, in which case the call to `strpos` is incorrect. This is why I saved it in the variable `$fqn`.
Also I find this very hard to read, I would just do `preg_split('/->|::/', $fqn)[0]`.
But tbh I think if you don't want to handle the case of already having typed a prefix, the check at line 137|135 can just go away and this `else` branch is not needed - it exists only to filter by an already-typed property name.